src/client.cpp

```diff
@@ -131,6 +134,7 @@ ADDON_STATUS ADDON_Create(void* hdl, void* props)
     m_currentStatus = ADDON_STATUS_LOST_CONNECTION;
     return m_currentStatus;
   }
+  m_catchupController = &m_data->GetCatchupController();
```

Comment:
Is it guaranteed that the raw pointer stored in this member will be valid as long as you access it using the member?

Reply:
Yes, everywhere it is used has a check for m_data which is only deleted on addon destroy.

Reply:
Cool. Thought that you had checked that. Just wanted to be sure.